package.json

```diff
@@ -22,10 +22,4 @@
     "url": "https://github.com/giliyoffe/Fibbage-Diconium/issues"
   },
   "homepage": "https://github.com/giliyoffe/Fibbage-Diconium#readme",
-  "dependencies": {
-    "express": "^4.17.1"
-  },
-  "devDependencies": {
-    "concurrently": "^5.3.0"
```

Comment:
this is actually being used for the start dev command so obviously it cant be removed.